release-notes/opensearch-sql.release-notes-3.0.0.0-beta1.md

```diff
@@ -0,0 +1,77 @@
+Compatible with OpenSearch and OpenSearch Dashboards Version 3.0.0.0-beta1
+
+### Breaking Changes
+* Unified OpenSearch PPL Data Type ([#3345](https://github.com/opensearch-project/sql/pull/3345))
```

Comment:
include breaking changes from Calcite also. https://github.com/opensearch-project/sql/issues/3248#issuecomment-2753128782

Reply:
> include breaking changes from Calcite also. [#3248 (comment)](https://github.com/opensearch-project/sql/issues/3248#issuecomment-2753128782)

Moved #3473 and #3439 which causing breaking behaviors https://github.com/opensearch-project/sql/pull/3529/commits/5445d415976091f87e2d0d2f2d991d66206a00d5

Reply:
Breaking changes in v3 is here https://github.com/opensearch-project/sql/issues/3487

Reply:
Oh, seems they are all PRs here rather than issues. And do we need to call out the new doc here? docs/dev/intro-v3-engine.md

Reply:
> And do we need to call out the new doc here? docs/dev/intro-v3-engine.md

This PR only covers the release notes. The dev doc update can be tracked separately. However, the dev doc PR can be included in the release notes if it is merged by eod today.

Reply:
Already included in Documentation section.